### DISCUSSION TITLE
Tools or plugin for code coverage using junit.

### DISCUSSION THREAD

**Discussion by pradeipk** [EclipseNebula/nebula#504] [Q&A]
I am using  integrated development environment and graphical user interface based on the Eclipse platform. I want to test the java code with coverage report. Need some guidance on the testing tool/plugin or if we can run the Junit.

**Comment by lcaron** [comment]
Hi

Sorry for the very late answer. Do you still need some help ?

**Comment by pradeipk** [comment]
Yes, any help in this regard is highly appreciated.

**Reply by lcaron** [reply]
Hi

Sorry for this very late answer. Did you investigate on SWTBot ? https://projects.eclipse.org/projects/technology.swtbot
